jar-infer/scripts/android-jar.py

```diff
@@ -10,13 +10,17 @@
 (options, args) = parser.parse_args()
 if not os.path.isfile(options.config):
   sys.exit('Error! No configuration file at: '+options.config)
-configParser = ConfigParser.RawConfigParser()   
+configParser = ConfigParser.SafeConfigParser()
 configParser.read(options.config)
 ajars_dir = configParser.get('android-paths', 'aosp-out-dir')
-stubsjar = configParser.get('android-paths', 'android-jar-path')
-nullaway_dir = configParser.get('jar-infer-paths', 'nullaway-dir')
+stubs_jar = configParser.get('android-paths', 'android-stubs-jar')
```

Comment:
Any reason for the mixed casing? I see `configParser` in camel case and `stubs_jar` in lower case. Probably better for all variables to be on the same case. PEP8 ( https://www.python.org/dev/peps/pep-0008/ ) suggests lowercase_with_underscores for local variables, and since this is the first python script in this repo, maybe we can adopt that convention (@msridhar ?)